docs/features/building/helm-arguments.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Helm Arguments
+---
+
+## What are Helm Arguments?
+
+**Helm Arguments** are used to reconfigure the default configurations in **Helm Charts**. They are similar to environment variables. Each **branch** of a Custom Service KintoBlock will have their own Helm Arguments. It is also posisble to add multiple Helm Arguments to a Custom Service KintoBlock.
+
+The Helm Arguments value can be added on the **KintoBlock Manage** page or via **Configure Dependencies** button in Deployment Manage Page. It is recommended to assign a `Value` for each Helm Argument that is added to the Custom Service KintoBlock. 
```

Comment:
"It is recommended to assign a `Value` for each Helm Argument that is added to the Custom Service KintoBlock. "

Not recommended, you might want to leave it empty.
It's not env variables.

Reply:
Done.